wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -796,7 +796,11 @@ instance BuildableSafeGen Wallet where
     %" balance="%buildSafe sl
     %" }")
     walId
-    walName
+    -- TODO(parsons.matt): remove this when we find out *why* this is
+    -- causing it to fail.
+    --
+    -- https://iohk.myjetbrains.com/youtrack/issue/R120-4#comment=93-20855
+    (T.filter C.isAscii walName)
```

Comment:
@parsonsmatt @KtorZ Would you guys refresh me on whether or not on `develop` we eventually fixed this properly on the `log-warper` side?

If yes, @jcmincke could remove this "hotfix" (i.e. get rid completely of this patch, and revert to simply `walName`).

Reply:
We fixed the log-warper on develop, actually using a different (patched) tip for the package.
See https://github.com/input-output-hk/cardano-sl/pull/2843

Nevertheless, develop _also_ contains the hotfix setting the `LANG` & `LC_ALL` env vars. In theory, it shouldn't matter. Yet I am not sure we want to remove it at this point.